lib/topological_inventory/ansible_tower/parser/service_instance_node.rb

```diff
@@ -29,6 +31,16 @@ def parse_service_instance_node(node)
             }
           )
         )
+
+        credentials = instance_node[:credentials].to_a
+        if credentials.present?
```

Comment:
if `credentials` are always array, then `present?` is not necessary, because `each` can handle empty array

Reply:
done 👍